numpy/core/src/umath/ufunc_object.c

```diff
@@ -2865,35 +2865,44 @@ PyUFunc_Reduce(PyUFuncObject *ufunc, PyArrayObject *arr, PyArrayObject *out,
 
     /* Get the identity */
     identity = _get_identity(ufunc, &reorderable);
```

Comment:
You need to check if this returns `NULL` like it did before, because that means that `reorderable` wasn't computed either.

Reply:
That's already done below; I think. I changed `identity` to `initializer` everywhere below, so all checks are now on `initializer`.